examples/music.py

```diff
@@ -4,11 +4,59 @@
 import discord
 from discord.ext import commands
 import lavalink
-from lavalink.audio_events import DefaultEventAdapter
+from lavalink.audio_events import TrackResumeEvent, TrackExceptionEvent, TrackStartEvent, TrackPauseEvent, TrackEndEvent, \
+    TrackStuckEvent, AudioTrack, AbstractPlayerEventAdapter
 
 time_rx = re.compile('[0-9]+')
 
 
+class DefaultEventAdapter(AbstractPlayerEventAdapter):
```

Comment:
I assume due to the fact ctx is passed, that this must be created for every player. If we could somehow remove the need to instantiate this in the music cog - perhaps have a convenience function in Client that will automatically register a single event adapter to every created player, or alternatively set one adapter which all players call, and users can make use of the `player` parameter to access required data.

Also as an alternative to passing ctx, users can store a textchannel ID in the player's user data dict and retrieve it later to send messages to a channel. 

I dislike the idea of holding onto things for too long - a channel could be deleted while the player is playing and may throw issues later when trying to post.